x-pack/plugin/eql/qa/rest/src/javaRestTest/java/org/elasticsearch/xpack/eql/EqlTestCluster.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.eql;
+
+import org.elasticsearch.test.cluster.ElasticsearchCluster;
+import org.elasticsearch.test.cluster.local.distribution.DistributionType;
+
+public class EqlTestCluster {
+
+    public static ElasticsearchCluster getCluster() {
```

Comment:
I think long term what we probably want is to put this in `RemoteClusterAwareEqlRestTestCase` or some other common base class to avoid all this duplication.

Reply:
Oh yes. 
I tried to go down that path and quickly realised it was going to be a big change - my inclination is that the 3 separate projects should be merged into one codebase that has a few parameters, rather than copies of test classes.